src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -43,10 +44,12 @@ export const ContactPolicy = InjectAppServices(
     }, [isContactPolicyEnabled, dopplerContactPolicyApiClient, appSessionRef]);
 
     const submitContactPolicyForm = async (values, { setSubmitting }) => {
+      setSubmitDisabled(true);
       setFormSubmitted(false);
       try {
         const { success } = await dopplerContactPolicyApiClient.updateAccountSettings(values);
         if (!success) {
+          setSubmitDisabled(false);
```

Comment:
should this be in the `finally` block?

Reply:
no, only enabled in case of error, otherwise it remains disabled

Reply:
Ok. But, then it would be necessary to reload the page again to be able to correct the error and send the form again?

Reply:
no, the idea is to keep the latest changes

Reply:
How then can the user submit the form again if the button is disabled?

Reply:
The flow is as follows:

- By default the Save button is disabled.
- The user modifies some value in the inputs or the switch.
- The save button is enabled.
- Changes are saved (The button is disabled while saving and the button is animated)
    - If an error occurs, the button is enabled so that the user can try again.
    - If it is saved successfully, the button remains disabled until the user modifies a value.